woocommerce/woocommerce-ios
# [Better Customer Selector] Hide the Address Form Search button everywhere

## Description

Before the Better Customer Selector feature we always showed a search button in the address form. As now we show the list directly, this is not necessary anymore. While we already hid the button when editing the order details, there's still the chance of editing the address on the order, and we didn't hide it there. With this PR we directly rely on the view model to ascertain whether it should be hidden or not.

## Testing instructions


With the `betterCustomerSelectionInOrder` feature flag enabled.

1. Go to orders
2. Tap on an order with a customer
3. Tap on View Billing Information
4. Tap on the Edit button for Billing Information
5. See that there's no search button in the Billing Address Form Navigation Bar

## Screenshots


## Before

https://github.com/woocommerce/woocommerce-ios/assets/1864060/60deb04c-01f4-4896-9ca7-db40102ca0c6

## After

https://github.com/woocommerce/woocommerce-ios/assets/1864060/e0933b94-97d3-484f-afa4-c891b24ba1d1

---
